test/PowerShellEditorServices.Test.Protocol/Server/OutputDebouncerTests.cs

```diff
@@ -76,8 +77,8 @@ public async Task OutputDebouncerDoesNotDuplicateOutput()
 
             // Ensure that the two events start with the correct lines
             Assert.Equal(2, messageSender.OutputEvents.Count);
-            Assert.Equal("Output 1", messageSender.OutputEvents[0].Output.Split('\r')[0]);
-            Assert.Equal("Output 26", messageSender.OutputEvents[1].Output.Split('\r')[0]);
+            Assert.Equal("Output 1", messageSender.OutputEvents[0].Output.Split('\n')[0].Trim('\r'));
```

Comment:
could we just do `Environment.NewLine` here instead?

Reply:
I looked into this, net461 doesn't mind this, but netcoreapp2.1 does for some reason...